### PR TITLE
Add qa tier

### PR DESCRIPTION
**Pull request by ktp242**
This PR sets the configurations for adding the QA tier. The changes include adding config/var_qa.env and update the deploy-qa scripts in package.json. Along with the documentations and the version.

Please see the new CheckoutRequestService-qa Lambda here
https://console.aws.amazon.com/lambda/home?region=us-east-1#/functions/CheckoutRequestService-qa?newFunction=true&tab=graph

and new checkout-requests-qa RDS here
https://console.aws.amazon.com/rds/home?region=us-east-1#dbinstance:id=checkout-requests-qa

They are both under nypl-digital-dev

### CODE REVIEW

**Review by emu47 — COMMENTED** [review]
Might remove the var_qa.env file from source control? Things like this are better left locally, I think.

**Comment by emu47** [comment]
Otherwise looks good to me.

**Review by nonword — APPROVED** [review]

**Comment by ktp242** [comment]
@emu47 I was thinking the same thing but I realized that we have to push var_qa.env to the remote so Travis can use the configurations to test/deploy the lambda service. I have also confirmed this with @kfriedman . I think at the moment the better way to execute it is to encrypt the sensitive info in .env

**Comment by ktp242** [comment]
I have updated the subnets and security groups to scripts for deployment. Also the settings on our Lambda instances.

**Comment by ktp242** [comment]
This has finally been deployed to QA! It should be all good for testing now. Please test with the URL https://qa-platform.nypl.org/api/v0.1/checkout-requests

I use the the POST body as below, but other valid data should work too
```
{
  "patronBarcode": "23333080894825",
  "itemBarcode": "33433000829147",
  "owningInstitutionId": "NYPL",
  "desiredDateDue": "2016-04-12T04:00:00.000-08:00"
}
```

A successful response could be as below,

```
{
    "data": {
        "id": 2,
        "success": true,
        "processed": true,
        "updatedDate": "2018-09-20T14:17:50-04:00",
        "createdDate": "2018-09-20T14:17:49-04:00",
        "patronBarcode": "23333080894825",
        "itemBarcode": "33433000829147",
        "owningInstitutionId": "NYPL",
        "desiredDateDue": "2016-04-12T04:00:00-08:00",
        "cancelRequestId": null,
        "jobId": "545ba3e44da78f5"
    },
    "count": 1,
    "statusCode": 208,
    "debugInfo": []
}
```

File: inputfile.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+w6Ar1jDW
```
Comment (nonword) [inline inputfile.txt]: This seems to be a db password? We should change it.

Comment (ktp242) [reply inputfile.txt]: Gosh! Sorry I accidentally included it. I am changing a new password now.

**Comment by ktp242** [comment]
The password to the database is updated and the files incorrectly included are removed as well.

**Pull request merged**